Makefile

```diff
@@ -351,18 +354,22 @@ _build/poplog_base/pop/pop/newpop.psv: _build/Stage1.proxy
         && (cd $$popsys; $$popsys/corepop %nort ../lib/lib/mkimage.p -entrymain ./newpop.psv ../lib/lib/newpop.p)
 
 # This target ensures that we have an unpacked base system with a valid corepop file.
-_build/Corepops.proxy: _build/Base.proxy _build/Corepops.Downloaded.proxy
+_build/Corepops.proxy: _build/Base.proxy _download/Corepops.Downloaded.proxy
+	cp -r _download/Corepops _build/
```

Comment:
Should be `cp -rpP` I think. The `-p` is to preserve permissions and important to protect against weird umask settings. And `-P` is just a good idea but actually doesn't matter as there are no symlinks in the `Corepops` repo.

Reply:
Done.